CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
 
+## [0.10.1] — January 4, 2024
+
+### Fixed
```

Comment:
If you have push permissions, you can use #76 and only need the changelog content and none of the version bumps.

Most of the time, I don't expect my changelogs to reviewed so I just have `cargo release` push it with the version bump, rather than creating a PR